pom.xml

```diff
@@ -0,0 +1,226 @@
+    </repository>
+  </distributionManagement>
+  
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    
+    <java.compiler.version>1.7</java.compiler.version>
+    <slf4j-api.version>1.7.7</slf4j-api.version>
+    
+    <logback-classic.version>1.1.2</logback-classic.version>
+    <selenium-java.version>2.43.1</selenium-java.version>
+    <sauce_testng.version>2.1.9</sauce_testng.version>
+    <testng.version>6.8</testng.version>
+    <mockito-all.version>1.8.4</mockito-all.version>    
+    <maven-compiler-plugin.version>3.1</maven-compiler-plugin.version>
+    <maven-surefire-plugin.version>2.17</maven-surefire-plugin.version>
+  </properties>
+  
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>org.slf4j</groupId>
+        <artifactId>slf4j-api</artifactId>
+        <version>${slf4j-api.version}</version>
+      </dependency>
+      <dependency>      
+        <groupId>ch.qos.logback</groupId>
+        <artifactId>logback-classic</artifactId>
+        <version>${logback-classic.version}</version>
+        <scope>test</scope>
+      </dependency>       
+      <dependency>
+        <groupId>org.seleniumhq.selenium</groupId>
+        <artifactId>selenium-java</artifactId>
+        <version>${selenium-java.version}</version>        
+      </dependency>
+      <dependency>
+        <groupId>com.saucelabs</groupId>
+        <artifactId>sauce_testng</artifactId>
+        <version>${sauce_testng.version}</version>         
+      </dependency>
+      <dependency>
+        <groupId>org.testng</groupId>
+        <artifactId>testng</artifactId>
+        <version>${testng.version}</version>      
+      </dependency>
+      <dependency>
+        <groupId>org.mockito</groupId>
+        <artifactId>mockito-all</artifactId>
+        <version>${mockito-all.version}</version>
+      </dependency>          
+      
+    </dependencies> 
+  </dependencyManagement>
+  
+  <dependencies>               
+    <dependency>
+      <groupId>org.slf4j</groupId>
+      <artifactId>slf4j-api</artifactId>      
+    </dependency>
+    <dependency>      
+      <groupId>ch.qos.logback</groupId>
+      <artifactId>logback-classic</artifactId>      
+      <scope>test</scope>
+    </dependency>    
+    <dependency>
+      <groupId>org.seleniumhq.selenium</groupId>
+      <artifactId>selenium-java</artifactId>            
+    </dependency>
+    <dependency>
+      <groupId>com.saucelabs</groupId>
+      <artifactId>sauce_testng</artifactId>           
+    </dependency>               
+    <dependency>
+      <groupId>org.testng</groupId>
+      <artifactId>testng</artifactId>      
+    </dependency>
+    
+    <dependency>
+      <groupId>org.mockito</groupId>
+      <artifactId>mockito-all</artifactId>      
+    </dependency>                       
+       
+  </dependencies>
+  <build>    
+    <!--    <finalName>web-test-framework</finalName>-->

```

Comment:
I recommend not checking in commented out code and leave comment blocks for strictly comments.